### PR TITLE
main

### PR DESCRIPTION
**Pull request by Burash1**

### CODE REVIEW

File: fibo.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    fib1 = 1
+    fib2 = 1
+
+    i = 0
+    while i < n - 2:
+        fib_sum = fib1 + fib2
+        fib1 = fib2
+        fib2 = fib_sum
+        i = i + 1
+    return fib2
```
Comment (Lijamaija) [inline fibo.py]: this function not needed for our project, we will have to delete it

File: main.py [hunk 2]
```diff
@@ -35,10 +49,13 @@ def main():
...
+toc = time.perf_counter()     # stop timer
 
 
 
 # Press the green button in the gutter to run the script.
 
 if __name__ == '__main__':
-    main()
\ No newline at end of file
+    print(f"Вычисление заняло {toc - tic:0.4f} секунд")
```
Comment (Lijamaija) [inline main.py:59]: make it English, as we write everything in English

File: main.py [hunk 3]
```diff
@@ -8,21 +8,35 @@
 # Logging <- HOMEWORK 24.09.2021
 # Timer - how much time it took < HOMEWORK 24.09.2021
 
+import logger
+logger.basicConfig(level=logger.DEBUG)
+
+import time     # timer
+tic = time.perf_counter()  # start timer
+
+from fibo import fibo   # set of tasks, To check the timer
```
Comment (Lijamaija) [inline main.py:17]: not needed since it's working

Comment (Burash1) [reply main.py:17]: fibo deleted

File: main.py (hunk 2 above)
Comment (Burash1) [reply main.py:59]: OK

File: fibo.py (hunk 1 above)
Comment (Burash1) [reply fibo.py]: deleted

**Pull request merged**